.github/workflows/nightly_wheels_build_and_test.yml

```diff
@@ -0,0 +1,47 @@
+name: Nightly Wheels Build and Test
+
+on:
+  pull_request:
+    paths:
+    - '.github/workflows/nightly_wheels_build_and_test.yml'
+    - 'scripts/wheels/**'
+
+
+  schedule:
+    - cron:  '0 1 * * *'
+
+  workflow_dispatch:
+
+# for cancelling redundant runs
+concurrency:
+  group: nightly-wheels-build-test-${{ github.head_ref }}
+  cancel-in-progress: true
+
+jobs:
+  linux-wheel:
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Build
+      shell: bash
+      run: |
+        python3 --version
+        mkdir ${GITHUB_WORKSPACE}/wheels
+        ls -al ${GITHUB_WORKSPACE}/wheels
+        echo "${GITHUB_HEAD_REF}"
+        docker run --rm -v ${GITHUB_WORKSPACE}/wheels:/data_swap_guest kratosmultiphysics/kratos-wheelbuilder-linux ${GITHUB_HEAD_REF}
+        ls -al ${GITHUB_WORKSPACE}/wheels
```

Comment:
I guess this has some debugging things leftover

Reply:
removed them apart from the last line, since this a convenient way to see which wheels have been successfully build